package.json

```diff
@@ -46,14 +46,17 @@
     "babel-core": "^6.26.0",
     "babel-eslint": "^8.2.6",
     "babel-loader": "^7.1.5",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
     "babel-preset-env": "^1.6.1",
     "browser-sync": "^2.24.7",
     "browserslist": "^4.1.0",
     "caniuse-db": "^1.0.30000878",
     "chalk": "^2.4.1",
     "del": "^3.0.0",
     "eslint": "^5.4.0",
+    "eslint-config-wordpress": "^2.0.0",
```

Comment:
There is also [@wordpress/eslint-plugin](https://www.npmjs.com/package/@wordpress/eslint-plugin) which is worth to look.